README.md

```diff
@@ -112,7 +112,24 @@ rust_library(name, srcs, crate_root, deps, data, crate_features, rustc_flags)
           if <code>srcs</code> contains only one file.
         </p>
       </td>
-    </td>
+    </tr>
+    <tr>
+      <td><code>crate_type</code></td>
+      <td>
+        <code>String, optional</code>
+        <p>
+          The type of crate to be produced during library compilation. This
+          list closely matches Cargo's own notion of crate-type, and the
+          available options are "lib", "rlib", "dylib", "cdylib", "staticlib",
+          and "proc-macro".
```

Comment:
This isn't evaluated as the required crate dependencies are impractical to import without `cargo-raze`.

For the record, the required crates are
```
extern crate proc_macro;
extern crate syn;
extern crate quote;
```